src/Neo/Neo.csproj

```diff
@@ -27,6 +27,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Neo.SmartContract.Testing" />
     <InternalsVisibleTo Include="Neo.SmartContract.TestEngine" />
+    <InternalsVisibleTo Include="Neo.Plugins.RpcServer.Tests" />
```

Comment:
@Jim8y 

Why just RpcServer here?

Reply:
adding ut for these plugin will be a very huge task, now i can only focus on rpcserver. whoever want to work on other plugin can add by urself.

Reply:
I mean, why not include `[Neo.Plugins.OracleService.Tests](https://github.com/neo-project/neo/tree/master/tests/Neo.Plugins.OracleService.Tests)` and [Neo.Plugins.Storage.Tests](https://github.com/neo-project/neo/tree/master/tests/Neo.Plugins.Storage.Tests)

What was this need?